framework/components/ecl-tables/_tables.scss

```diff
@@ -84,32 +39,6 @@
 
 @media (min-width: map-get($ecl-grid-breakpoints, 'md') + 1) {
```

Comment:
`@media (min-width: map-get($ecl-grid-breakpoints, 'md') + 1)` should be replaced by `@include ecl-media-breakpoint-up(md)`